src/Symfony/Component/Security/Tests/Http/Session/SessionAuthenticationStrategyTest.php

```diff
@@ -39,6 +39,10 @@ public function testUnsupportedStrategy()
 
     public function testSessionIsMigrated()
     {
+        if (PHP_VERSION_ID >= 50400 && PHP_VERSION_ID < 50411) {
+            $this->markTestSkipped('We must not destroy the old session on php 5.4.0 - 5.4.10.');
```

Comment:
Keeping two tests is fine by me.

@derrabus Can you update this PR so that I can merge it today and start releasing 2.6.2? Thank you very much.